scripts/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - bash
       - -c
       # https://docs.microsoft.com/en-us/sql/relational-databases/logs/control-transaction-durability?view=sql-server-ver15#bkmk_DbControl
-      - 'until /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P S0meVeryHardPassword -d master -Q "CREATE DATABASE knex_test; ALTER DATABASE knex_test SET DELAYED_DURABILITY = FORCED"; do sleep 5; done'
+      - 'until /opt/mssql-tools/bin/sqlcmd -S mssql -U sa -P S0meVeryHardPassword -d master -Q "CREATE DATABASE knex_test; ALTER DATABASE knex_test SET ALLOW_SNAPSHOT_ISOLATION ON; ALTER DATABASE knex_test SET DELAYED_DURABILITY = FORCED"; do sleep 5; done'
```

Comment:
This can't be applied in runtime using knex.raw?

Reply:
I tried this, but it threw an error, I guess there's side effects elsewhere
`ALTER DATABASE knex_test SET SNAPSHOT ON WITH ROLLBACK IMMEDIATE`
https://stackoverflow.com/a/1253452/6725059